addie/processing/mantid/master_table/table_tree_handler.py

```diff
@@ -795,14 +813,24 @@ def rows_remove(self):
 
     def rows_duplicate(self):
         '''duplicate currently selected rows'''
-        o_table_row = TableRowHandler(main_window=self.main_window)
-        o_table_row.insert_blank_row()
+        # o_table_row = TableRowHandler(main_window=self.main_window)
+        # o_table_row.insert_blank_row()
```

Comment:
Same, let's just delete out the old commented out code. We have it in version control if we need it again.